index.js

```diff
@@ -71,6 +71,16 @@ module.exports = function(props) {
 
   return function(req, res, next) {
     var language;
+    var queryName = props.queryName || 'language';
+    var queryLanguage = req.query[queryName];
+
+    if(typeof queryLanguage === 'string' && queryLanguage.length > 1) {
```

Comment:
Can you add a check that the language is supported too? `props.languages.indexOf(queryLanguage) !== -1`.

Also add a corresponding test for it.